README.md

```diff
@@ -7,6 +7,19 @@ and use of prometheus at scale (when you have more than one prometheus host).
 Promxy delivers this unified access endpoint without requiring **any** sidecars,
 custom-builds, or other changes to your prometheus infrastructure.
 
+## Why promxy?
+[**Detailed version**](https://github.com/jacksontj/promxy/blob/master/STORY.md)
```

Comment:
a few thoughts here:

s/story/motivation/ ?

Also I think you can just link relatively, so docs will work in forks:

```markdown
[Detailed Version](STORY.md)
```